core/domain/classifier_services.py

```diff
@@ -305,9 +313,9 @@ def _update_classifier_training_job(classifier_training_job_model, status):
     classifier_training_job_model.put()
 
 
-def save_classifier_training_job(algorithm_id, exp_id, exp_version,
-                                 state_name, status, training_data,
-                                 job_id="None"):
+def save_classifier_training_job(algorithm_id, exp_id,
+                                 exp_version, state_name, training_data,
+                                 status=None, job_id=None):
```

Comment:
Can we drop the default args? So: (..., training_data, status, job_id)

I thought I commented on this earlier...